ospfd/ospf_asbr.c

```diff
@@ -1159,7 +1159,8 @@ static void ospf_external_aggr_timer(struct ospf *ospf,
 					"%s, Restarting Aggregator delay timer.",
 					__func__);
 			EVENT_OFF(ospf->t_external_aggr);
-		}
+		} else
```

Comment:
this is too general. why not add a clause for repeated AGGR_DEL like the one that's already present for AGGR_ADD?

Reply:
No. IMO "DEL" is not same to "ADD".
The continuous "DEL"s come, the first "DEL" should do the real thing: start timer and delete the item.

Reply:
hmm - that's exactly what I wrote, but you said "no".
So: there's a clause for ADD that says "don't re-start the timer if it's already running for ADD". I suggest doing the same for DEL : if the timer is already running for DEL, emit a debug and don't reset the timer.

Reply:
I misunderstood your meaning just now.
I got your point, and re-pushed one. Thanks for your review.